docs/fidesops/docs/guides/storage.md

```diff
@@ -95,7 +95,7 @@ Example response:
                 "naming": "request_id",
                 "object_name": "requests"
             },
-            "key": "s3-storage-2"
+            "key": "s3_storage_2"
```

Comment:
Also L43 in this same doc has type hinting in a sample payload and uses string instead of fideskey

```json
  {
    "destinations": [
      {
        "name": str,
        "key": str (optional),
        "type": str,
        "format": str
        "details": {
          # s3
          "bucket": str,
          "naming": str,
          # onetrust
          "service_name": str,
          "onetrust_polling_hr": int,
          "onetrust_polling_day_of_week": int
        }
      }
    ]
  }
```